src/components/List.jsx

```diff
@@ -29,6 +30,14 @@ export const List = ({ token }) => {
     });
   };
 
+  const deleteItem = async (document) => {
+    if (
+      window.confirm(`Are you sure you want to delete ${document.data().item}?`)
```

Comment:
I love inconsistent JS can be... one would assume you have to treat this like a promise but yet here we are